### ISSUE TITLE
test_hyphens_find causes page to scroll to the top

### ISSUE THREAD

**Issue by entioentio** [Modernizr/Modernizr#2193]
This line:

    result = window.find(testword + testword);

Makes browser (at least chrome) to highlight the text, which results in page jumping to the top. My initial idea is to save window position and revert it after the check, but it seems wrong. Any better ideas anyone?

**Comment by adoerler** [comment]
Hi,

today we came across the very same issue.
I've found out that the line causing the page scrolling to top is 157 in hyphens.js:
https://github.com/Modernizr/Modernizr/blob/5eea7e2a213edc9e83a47b6414d0250468d83471/feature-detects/css/hyphens.js#L157

i.e. `dummy.focus();`

In our case we have an website using modernizer and css-hyphens detection enabled.
Inside this site wie have a LightCase lightbox which loads a form generated by the same CMS and therefore modernizr runs inside this iframe too.
Setting the focus to the dummy element makes Chrome (at least recent versions) to scroll to the focused element (although this element is not visible at all).

Our workaround is to remove the hyphens detection when the page is loaded in our lightbox, but a more general solution would be great.

Kind regards
Andreas
